src/content/en/fundamentals/media/_code/tiny-fullscreen-shim.js

```diff
@@ -0,0 +1,21 @@
+HTMLElement.prototype.requestFullscreen = HTMLElement.prototype.requestFullscreen || HTMLElement.prototype.webkitRequestFullscreen;
```

Comment:
This is missing the moz variants. I would strongly encourage you to not attempt to shim the unprefixed API, and instead write helper methods that tries to use each API in order, with fallbacks. https://github.com/sindresorhus/screenfull.js is the best such library I have found, and is already used in the wild, so using that does not risk placing any additional compat constraints on the unprefixed API before it has shipped.

Note that requestFullscreen should be on Element and exitFullscreen on Document, not HTMLElement/HTMLDocument.